java-extensions/hudi-reader/src/main/java/com/starrocks/hudi/reader/HudiColumnValue.java

```diff
@@ -75,7 +75,7 @@ public double getDouble() {
     }
 
     @Override
-    public String getString() {
+    public String getString(ColumnType.TypeValue type) {
```

Comment:
how this thing pass compiler check?

Or do we have another method `getString()` ?

Reply:
The signature of `getString()` has changed in this PR.